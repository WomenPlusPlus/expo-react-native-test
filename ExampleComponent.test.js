import React from 'react';
import renderer from 'react-test-renderer';

import ExampleComponent from './ExampleComponent';

describe('<ExampleComponent />', () => {
  it('handles props correctly', () => {
    const testComponent  = renderer.create(<ExampleComponent testID="ex" arg={4} />)
    expect(testComponent.root.props.arg).toBe(4)
    expect(testComponent.toJSON()).toMatchSnapshot()
  });
});

