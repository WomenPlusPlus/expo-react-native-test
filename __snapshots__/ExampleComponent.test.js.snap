// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExampleComponent /> handles props correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Child component
  </Text>
  <Text>
    4
  </Text>
</View>
`;
